src/components/DownloadCards/DownloadCard/index.tsx

```diff
@@ -31,6 +31,7 @@ export default function DownloadCard({
       key={name}
       role="presentation"
       onClick={handleSelectCard}
+      tabIndex={0}
```

Comment:
@marcustisater @benhalverson  
It looks like eslint complains for that, according to esling rule https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-tabindex.md
The question here is, do we anyway want to go with the approach of "dotted-border" on each card?
because we could wrap each `<li>` element in an interactive element like button. however, it sounds quite unconventional. what do you think?